v20.1/import.md

```diff
@@ -309,6 +311,59 @@ CSV DATA (
 );
 ~~~
 
+#### Using a wildcard
+
+You can specify file patterns to match instead of explicitly listing every file. Use the `*` wildcard character to include matching files directly under the specified path. A wildcard can be used to include:
+
+- All files in a given directory (e.g.,`s3://bucket-name/path/to/data/*`)
+- All files in a given directory that end with a given string (e.g., `s3://bucket-name/files/*.csv`)
+- All files in a given directory that start with a given string (e.g., `s3://bucket-name/files/data*`)
+-All files in a given directory that start and end with a given string (e.g., `s3://bucket-name/files/data*.csv`)
+
```

Comment:
paths are matched using https://golang.org/pkg/path/filepath/#Match if we want to link to it for more details.